### DISCUSSION TITLE
writeups/hackthissite/basic-8/

### DISCUSSION THREAD

**Discussion by giscus** [Aftab700/aftab700.github.io#4] [Announcements]
# writeups/hackthissite/basic-8/

The password is yet again hidden in an unknown file. Sam's daughter has begun learning PHP, and has a small script to demonstrate her knowledge. Requirements: Knowledge of SSI (dynamic html executed by the server, rather than the browser)

https://aftabsama.com/writeups/hackthissite/basic-8/

**Comment by faizy-cyber** [comment]
but can u tell the method that will not approach the ssi
